abot.go

```diff
@@ -71,7 +71,15 @@ func main() {
 					Aliases: []string{"i"},
 					Usage:   "download and install plugins listed in plugins.json",
 					Action: func(c *cli.Context) {
-						installPlugins()
+						if err := installPlugins(); err != nil {
```

Comment:
We'd want to delete the `plugins.go` file that we created in `installPlugins` in this err block. We should also check the value returned from `os.Remove("plugins.go")` to ensure we ignore `err == os.ErrNotExist`, as that's fine (it simply means we errored out before the file was created).